integration/tests/v2_to_v3_upgrade.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# this test script:
+# 1. deploys an older version of Automate and upgrades it to v2 using the beta CLI,
```

Comment:
please modify the comments in line with the test. seems these are not modified after copying

Reply:
done